### PR TITLE
Update raw keyboard example

### PR DESCRIPTION
**Pull request by abarth**
We now center the body correctly and handle Fuchsia key events.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: dev/manual_tests/raw_keyboard.dart [hunk 1]
```diff
@@ -49,28 +49,29 @@ class _HardwareKeyDemoState extends State<RawKeyboardDemo> {
         onTap: () {
```
Comment (Hixie) [inline dev/manual_tests/raw_keyboard.dart]: The reason I called it "at" was so you could do `if (Focus.at(context)) ...`, so it's sad to see you factor that out into a separate variable. :-P

**Comment by Hixie** [comment]
LGTM

File: dev/manual_tests/raw_keyboard.dart (hunk 1 above)
Comment (abarth) [inline dev/manual_tests/raw_keyboard.dart]: Yeah, it's used in two places, unfortunately.

**Pull request merged**